docs/02-app/02-api-reference/06-cli/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: CLI
+description: API Reference for the Next.js Command Line Interface (CLI) tools.
+---
+
+{/* The content of this doc is shared between the app and pages router. You can use the `<PagesOnly>Content</PagesOnly>` component to add content that is specific to the Pages Router. Any shared content should not be wrapped in a component. */}
+
+Next.js comes with **two** Command Line Interface (CLI) tools:
+
+- **`create-next-app`**: Quickly create a new Next.js application using the default template or an [example](https://github.com/vercel/next.js/tree/canary/examples) from a public Github repository.
+- **`next`**: Run the Next.js development server, build your application, and more.
```

Comment:
Should these have links to the relevant docs?

Reply:
They are linked below automatically :)